opm/input/eclipse/Python/PyRunModule.cpp

```diff
@@ -54,10 +54,10 @@ PyRunModule::PyRunModule(std::shared_ptr<const Python> python, const std::string
     try {
         this->opm_embedded = py::module::import("opm_embedded");
     } catch (const std::exception& e) {
-        OpmLog::error(fmt::format("Exception thrown when loading Python module opm_embedded: {}"), e.what());
+        OpmLog::error(fmt::format("Exception thrown when loading Python module opm_embedded: {}. Possibly the PYTHONPATH of the system is not set correctly.", e.what()));
```

Comment:
@blattms: Here, one bracket was not set correctly. I've corrected that now and added a sentence about the PYTHONPATH.